SuperBuild/External_OpenSSL.cmake

```diff
@@ -91,6 +92,14 @@ set(ENV{VS_UNICODE_OUTPUT} \"\")
   # This is required when building Slicer using a cronjob where the
   # default environment is restricted.
   set(ENV{PATH} \"/usr/sbin:\$ENV{PATH}\")
+
+  # Allow the SDKROOT to be found on mac when
+  # building OpenSSL. This is needed to find
+  # standard header files and avoid errors like
+  # | ./cryptlib.h:62:11: fatal error: 'stdlib.h' file not found
+  # | # include <stdlib.h>
+  # |
+  set(ENV{SDKROOT} \"${CMAKE_OSX_SYSROOT}\")
```

Comment:
I haven't had a chance to test it yet, but I think this solution is more general so you wouldn't need to set it here.

https://github.com/Slicer/Slicer/issues/4681#issuecomment-604618256

Reply:
@pieper Thanks I did not see the larger issue.  If I get a chance, I'll try to get that known fix in place.  I spent a long time tracking down what the error was, but I did not see the other issue message.

Reply:
Thanks for investigating @hjmjohnson , I just spent some time poking around but didn't find the best spot to set the environment.  Jc should know where to look.